variation-commons-beacon/src/main/java/uk/ac/ebi/eva/commons/beacon/models/BeaconAlleleRequest.java

```diff
@@ -21,7 +21,10 @@
 import javax.validation.constraints.NotNull;
 import java.util.List;
 
-public class BeaconAlleleRequestBody {
+/**
+ * Allele request against a beacon. The allele must be mapped against a sequence in a genome, and a range can be optionally specified.
+ */
+public class BeaconAlleleRequest {
 
     @NotNull(message = "referenceName should not be null")
```

Comment:
```suggestion
    @NotNull(message = "A reference name must be provided")
```

Same with `reference bases` and `assembly identifier`.